src/core/AudioEngine.cpp

```diff
@@ -463,7 +454,7 @@ void AudioEngine::swapBuffers()
 	m_inputBufferFrames[m_inputBufferWrite] = 0;
 
 	std::swap(m_outputBufferRead, m_outputBufferWrite);
-	BufferManager::clear(m_outputBufferWrite, m_framesPerPeriod);
+	std::fill(m_outputBufferWrite.begin(), m_outputBufferWrite.end(), 0);
```

Comment:
Replace this with `std::fill_n(m_outputBufferRead.get(), m_framesPerPeriod, surroundSampleFrame{});`

Reply:
done